packages/orbit-components/src/NotificationBadge/NotificationBadge.stories.tsx

```diff
@@ -8,238 +9,78 @@ import NotificationBadge from ".";
+  argTypes: {
+    icon: {
+      options: Object.keys(Icons),
+      control: {
+        type: "select",
+      },
+    },
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
   },
 };
 
-Dark.args = {
-  content: "10",
-};
-
-export const White = ({ content }) => {
-  return (
-    <div style={{ backgroundColor: "#46515e", padding: "10px" }}>
-      <NotificationBadge type={TYPE_OPTIONS.WHITE}>{content}</NotificationBadge>
-    </div>
-  );
-};
-
-White.story = {
-  parameters: {
-    info: "Check Orbit.Kiwi for more detailed design guidelines.",
-  },
-};
+export default meta;
 
-White.args = {
-  content: "10",
-};
+type Story = StoryObj<typeof NotificationBadge>;
 
-export const Playground = ({ content, type, icon, dataTest, ariaLabel }) => {
-  const Icon = getIcon(icon);
-
-  return (
-    <NotificationBadge
-      type={type}
-      dataTest={dataTest}
-      ariaLabel={ariaLabel}
-      icon={Icon && <Icon />}
-    >
-      {content}
-    </NotificationBadge>
-  );
-};
+export const Default: Story = {
+  render: ({ children }) => <NotificationBadge>{children}</NotificationBadge>,
 
-Playground.story = {
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["icon", "type", "ariaLabel"],
+    },
   },
 };
 
-Playground.args = {
-  content: "10",
-  type: TYPE_OPTIONS.INFO,
-  dataTest: "test",
-  ariaLabel: "additional information for screen readers",
-  icon: "Airplane",
-};
+export const Playground: Story = {
+  render: ({ children, icon, ...args }) => {
+    const Icon = typeof icon === "string" && getIcon(icon);
 
-Playground.argTypes = {
-  type: {
-    options: Object.values(TYPE_OPTIONS),
-    control: {
-      type: "select",
-    },
+    return (
+      <NotificationBadge {...args} icon={Icon && <Icon />}>
+        {children}
+      </NotificationBadge>
+    );
   },
-  icon: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
-    },
-  },
-};
 
-export const Accessibility = ({ content, ariaLabel }) => {
-  return (
-    <NotificationBadge type="info" ariaLabel={ariaLabel}>
-      {content}
-    </NotificationBadge>
-  );
-};
-
-Accessibility.story = {
   parameters: {
-    info: "This is a preview of component accessibility props",
+    info: "Playground of NotificationBadge. Try all of possible options. Check Orbit.Kiwi for more detailed design guidelines.",
   },
 };
 
-Accessibility.args = {
-  content: "10",
-  ariaLabel: "additional information for screen readers",
-};
-
-export const Rtl = () => (
-  <RenderInRtl>
-    <NotificationBadge type="info">10</NotificationBadge>
-  </RenderInRtl>
-);
-
-Rtl.story = {
-  name: "RTL",
+export const Rtl: Story = {
+  render: () => (
+    <RenderInRtl>
+      <NotificationBadge type="info">10</NotificationBadge>
```

Comment:
Why are props here static and not spread here as args? And same for Default story actually.

Reply:
RTL:
We can include args there, I wanted to keep the basic minimal setup for RTL version. If we would like to go with `args` spreading in the code, I need to do an "additional" setup, like the args definition
```
args: {
  icon: undefined,
},
```

and defining `children` in render and child component (as it's in Playground story right now).
```
render: ({ children, ...args }) => (
    <RenderInRtl>
      <NotificationBadge {...args}>{children}</NotificationBadge>
      ...
```

from my point of view, I think that a minimal setup could be enough just for RTL demonstration and have to code as simple as possible.


**default story:**
all of the props (args) are not mandatory, I kept `children` to demonstrate that the NotificationBadge doesn't have to include _only_ icons.

Reply:
> If we would like to go with args spreading in the code, I need to do an "additional" setup

But why? If this is the default, then we can just use this default, no?

```
args: {
    children: "10",
    icon: "null",
    type: TYPE_OPTIONS.INFO,
    ariaLabel: "additional information for screen readers",
  },
```

And if the `icon` is an issue, it can be set to `undefined` on meta, no?

I am starting to get lost when do we want to use args and we are just hardcoding :D